editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1880,6 +1905,12 @@ bool CanvasItemEditor::_gui_input_scale(const Ref<InputEvent> &p_event) {
 					}
 				}
 			}
+
+			if (snap_scale == true && is_ctrl == false) {
```

Comment:
In general, we tend to be the less verbose possible. So I'd say we would generally use `if (snap_scale && !is_ctrl)`. But to be honest it is not that important. ^^